charts/htp-builder/Chart.yaml

```diff
@@ -12,6 +12,8 @@ keywords:
   - sampling
   - tracing
   - collector
+  - htp
+  - builder
```

Comment:
i wonder if builder is confusing in this context (with the existing ocb)

Reply:
```suggestion
```